OK, the test run of this showed its value, since the scanner is failing right now.
See [here](https://github.com/wolfi-dev/os/actions/runs/10511419863/job/29122530866?pr=26909), which shows:

> Unexpected error content
> | 2024/08/22 15:54:20 ERRO failed to create scanner: failed to load vulnerability database: unable to update vulnerability database: bad db checksum (/tmp/grype-scratch4276522467/vulnerability.db): "sha256:a11915b8368897dd446ad5cbb855414870297841f8bccc3b466c5c5e9ba5539b" vs "sha256:9aba357712f1f68620ccd19349038e59f17a337189a075f9abea884591925f9b"
> Error: Process completed with exit code 1.

--- a/.github/workflows/ci-build.yaml
+++ b/.github/workflows/ci-build.yaml
@@ -335,10 +335,21 @@ jobs:
 
       - name: Scan for CVEs
         run: |
+          stderrf="/tmp/scan.stderr"
           wolfictl scan \
             --build-log \
             --advisories-repo-dir 'data/wolfi-advisories' \
             --advisory-filter 'resolved' \
             --require-zero \
             /tmp/artifacts-1 \
-            2> /dev/null # The error message renders strangely on GitHub Actions, and the important information is already being sent to stdout.
+            2> "$stderrf" && rc=0 || rc=$?
+
+          # The error message renders strangely on GitHub Actions so it is
+          # captured and shown only if it does not match expected "vulnerabilities found"
+          if [ $rc -ne 0 ] &&
+             ! grep -qi "vulnerabilities found" "$stderrf"; then
+             echo "Unexpected error content" 1>&2
+             sed 's,^,| ,' "$stderrf" 1>&2
+          fi
+          rm -f "$stderrf"
+          exit $rc

